config/routes.rb

```diff
@@ -32,7 +32,7 @@
       patch 'viewed', to: 'paperworks#viewed', on: :member
     end
     resources :case_notes, only: [:show, :create, :update, :destroy] do
-      patch 'internal', to: 'case_notes#internal', on: :member
+      patch 'not_visible', to: 'case_notes#visible', on: :member
```

Comment:
this should say case_notes#not_visible since in the case notes controller you changed the function name to not_visible